Orange/widgets/settings.py

```diff
@@ -398,14 +398,17 @@ def write_defaults(self):
         filename = self._get_settings_filename()
         os.makedirs(os.path.dirname(filename), exist_ok=True)
 
-        settings_file = open(filename, "wb")
         try:
-            self.write_defaults_file(settings_file)
-        except (EOFError, IOError, pickle.PicklingError):
-            settings_file.close()
-            os.remove(filename)
-        else:
-            settings_file.close()
+            settings_file = open(filename, "wb")
+            try:
+                self.write_defaults_file(settings_file)
+            except (EOFError, IOError, pickle.PicklingError):
+                settings_file.close()
+                os.remove(filename)
+            else:
+                settings_file.close()
+        except PermissionError:
+            pass
```

Comment:
It might be a good idea to log the problem as it looks like this will happen every time.